.prettierrc

```diff
@@ -0,0 +1,14 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": true,
```

Comment:
Are there notable advantages of single quotes over double quotes? Visually, double quotes stand out more to me; they are consistent with JSON (which even this very file requires); and you don't have issues putting contractions like "don't" inside them.